11-payment-encoding.md

```diff
@@ -278,6 +281,37 @@ The field is big-endian.  The least-significant bit is numbered 0,
 which is _even_, and the next most significant bit is numbered 1,
 which is _odd_.
 
+Features are listed in [09-features.md](BOLT #9) [FIXME: Or will be after
+flattened features is merged!].
+
+Note that the `payment_secret` feature probing attacks from nodes
```

Comment:
```suggestion
Note that the `payment_secret` feature prevents probing attacks from nodes
```

Reply:
still unresolved?

Reply:
Ah, this was fixed in #712 which reworks this paragraph anyway.  Let me roll it in here, then I can rebase #712